v2/emailpassword/common-customizations/sessions/update-jwt-payload.mdx

```diff
@@ -62,7 +62,7 @@ let sessionHandles = Session.getAllSessionHandlesForUser(userId);
 
 // we update all the session's JWTs for this user
 sessionHandles.forEach(async (handle) => {
-      let currJWTPayload = await Session.getJWTPayload(handle);
+      let currJWTPayload = (await Session.getSessionInformation(handle)).userDataInJWT;
```

Comment:
I don't like `userDataInJWT`. Maybe it should be `(await Session.getSessionInformation(handle)).jwtPayload;`

Reply:
It does make sense but this will require a change in the node SDK and core (if we want to keep the naming consistent across). We could just make the change in the node SDK but that might introduce confusion

Reply:
You can make the change just to the node SDK.

Reply:
Done